spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/ReadIntegrationTestBase.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.cloud.spark.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.MathContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Encoders;
+import org.apache.spark.sql.Row;
+import org.junit.Test;
+
+public class ReadIntegrationTestBase extends SparkSpannerIntegrationTestBase {
+
+  private void testReadFromQueryInternal(String table, String query) {
+    Map<String, String> props = this.connectionProperties();
+    Dataset<Row> df =
+        spark
+            .read()
+            .format("cloud-spanner")
+            .option("viewsEnabled", true)
+            .option("projectId", props.get("projectId"))
+            .option("instanceId", props.get("instanceId"))
+            .option("databaseId", props.get("databaseId"))
+            .option("emulatorHost", props.get("emulatorHost"))
+            .option("table", table)
+            .load(query);
```

Comment:
Query is not an parameter of load. 
![image](https://github.com/GoogleCloudDataproc/spark-spanner-connector/assets/82131353/2b75347d-af07-432c-a7c6-5351ae66bb23)
 
Please remove the query.

Reply:
Done, thanks @halio-g: I was previously emulating from the BigQuery tests but now removed.